experimental/pgbouncer/pgbouncer/pgbouncer-deployment.yaml

```diff
@@ -15,37 +15,37 @@ spec:
         app: pgbouncer
     spec:
       containers:
-      - name: pgbouncer
-        image: pgbouncer/pgbouncer:1.11.0
-        env:
-          - name: PGBOUNCER_PASSWORD
-            valueFrom:
-              secretKeyRef:
-                name: pgxl-passwords-collection
-                key: pgpass
-        ports:
-        - containerPort: 5432
-        command:
-          - "sh"
-          - "-c"
-          - >
-            echo "
-              [databases]
-              * = host = db-pgb-postgres-xl-svc port=5432
+        - name: pgbouncer
```

Comment:
I see your appoach here and I think that we should start another chart to do this. If we can use an existing chart that would be better. Using echo in yaml commands is very unstable, and would cause production issues due to either errors in commit, encoding or just "junior" engineer work :) 

For now approved, but this would have to change if we ever move this to examples.

Reply:
ok thanks for letting me know. We can definitely change it at a later point.

Reply:
I'm happy to start a chart for it and help maintain it at some point if you too busy to do it. I will need to learn helm though so it will be some time. How do you suggest not using echo statements? config maps?

Reply:
oh I see. Helm is a lot more powerful than i realised. Looking at these docs to get an idea. https://docs.bitnami.com/kubernetes/how-to/create-your-first-helm-chart/ will do a chart for some experience in creating charts at a later point.

Reply:
Note that templeting here is also important, and should be implemented same as in this chart. If you dont mind waiting for a few weeks I would look at the other chars for pg_bouncer and see if we can use these.  Otherwise I don't mind creating the new chart with its basics and then add you as a code owner and manager to that. 

Otherwise feel free to add a new chart - I will join, though using an existing one is better practice if that's possible. 

It should be noted that at some point I plan to try and submit these charts to the general helm chart repository [here](https://github.com/helm/charts/tree/master/stable), so it must be able to fit within the parameters and rules for that repo.  Moreover, since there is no community blessed pg_bouncer helm chart, we can in general try and add both charts to the general community one and they will then be supported by them.

Reply:
ok brilliant that would be great to get this added at some point. ok thanks I will just use the deployment for now and see what you find. Reusing one would definitely be better. My biggest issues were finding one that supports auth_query as I'm doing in the examples and using a newer version of pgbouncer. If there is any chart out there that can do this or can be added that would be great.